incubator/kubevirt/templates/controller-service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "kubevirt.fullname" . }}-controller
+  namespace: {{ .Values.kubevirt.namespace }}
+  labels:
+{{ include "labels.standard" . | indent 4 }}
+    component: "virt-api-server"
+spec:
+  ports:
+    - port: 8182
+      targetPort: virt-controller
+  selector:
+{{ include "selectors.standard" . | indent 4 }}
```

Comment:
`chart` and `heritage` should not be part of the selector.

Reply:
fixed